apps/api/src/app/events/e2e/trigger-event.e2e.ts

```diff
@@ -2433,6 +2433,19 @@ describe('Trigger event - /v1/events/trigger (POST) #novu-v2', function () {
 
       it(`should not create multiple subscribers when multiple triggers are made        
          with the same not created subscribers `, async () => {
+        // Access subscriberRepository and print database indexes
+        console.log('process.env', process.env);
```

Comment:
In my PR about e2e fixes I removed this test now that we have an index in Mongo as there is no need to test the Mongo index. We trust it.